integration/rocket-chat/main.js

```diff
@@ -991,12 +989,11 @@ function SmartiWidget(element, _options) {
             } else {
                 showMsg({i18nObj: {code:'smarti.no-widgets'}});
             }
-        } else {
-            similarityQuery = getProp(similarityQueryPath, data);
-            $.each(widgets, (i, wgt) => {
-                wgt.refresh();
-            });
```

Comment:
Moving this code out of the else block will cause the widgets to load the initial data twice!